configure

```diff
@@ -12,15 +12,26 @@ else
 fi
 
 if [ -n "$xml2_include_dir" ]; then
-    PKG_CFLAGS="-I\"$xml2_include_dir\""
+    PKG_CFLAGS="$PKG_CFLAGS -I\"$xml2_include_dir\""
     echo "libxml2 include directory: $xml2_include_dir"
 else
     echo "Error: libxml2 include directory not found"
     exit 1
 fi
 
+OBJECTS='${SOURCES}'
+
+if echo '#include <glpk.h>' | $(R CMD config CC) -E $CPPFLAGS - > /dev/null 2>&1; then
+  echo "Using installed GLPK"
+  PKG_LIBS="$PKG_LIBS -lglpk"
+else
+  echo "Using vendored GLPK"
+  PKG_CFLAGS="$PKG_CFLAGS -Ivendor/cigraph/vendor/glpk -Ivendor/cigraph/vendor/glpk/env -Ivendor/cigraph/vendor/glpk/minisat -Ivendor/cigraph/vendor/glpk/misc -Ivendor/cigraph/vendor/glpk/draft -Ivendor/cigraph/vendor/glpk/npp -Ivendor/cigraph/vendor/glpk/api -Ivendor/cigraph/vendor/glpk/mpl -Ivendor/cigraph/vendor/glpk/bflib -Ivendor/cigraph/vendor/glpk/amd -Ivendor/cigraph/vendor/glpk/simplex -Ivendor/cigraph/vendor/glpk/colamd"
```

Comment:
`-I` flags should go in CPPFLAGS instead of CFLAGS, no?

Reply:
Is this comment just about naming? As I understand it doesn't has functional meaning.

Reply:
We can leave it as is and resolve later if needed.